nncf/experimental/openvino_native/graph/model_transformer.py

```diff
@@ -52,16 +53,7 @@ def __init__(self, model: ov.Model):
         :param model: OpenVINO model to be transformed.
         """
         super().__init__(model)
-        self._model = model.clone()
-        self.name_to_node_mapping = {op.get_friendly_name(): op for op in self._model.get_ops()}
```

Comment:
`self.name_to_node_mapping` stores the mapping between the node name and the corresponding node.
When we do transformations onto a model we use this mapping to get or update the node's information. 
As we want to update the node's information we need to store the mapping from the model on which transformation is applied, because of that it build for every cloned model